src/licenses/entities/license/activeSeatsFindResult.ts

```diff
@@ -3,56 +3,35 @@ import {
   FiltersParameters,
   SortParameters,
 } from '../../licensesClient';
-import { AbstractEntity } from '../../../abstractEntity';
+import {
+  AbstractActiveSeats,
+  ActiveSeatsData,
+  ActiveSeatsFields,
+} from '../../../common/abstract/abstractActiveSeats';
 
-export enum ActiveSeatsFindResultFields {
-  COLUMN_LAST_UPDATE = 'lastUpdate',
-  COLUMN_NUMBER = 'number',
-}
-
-export type ActiveSeatsFindresultData = {
-  [ActiveSeatsFindResultFields.COLUMN_LAST_UPDATE]?: string | null;
-  [ActiveSeatsFindResultFields.COLUMN_NUMBER]?: number | null;
-};
+export type ActiveSeatsFindResultData = ActiveSeatsData;
 
 export type ActiveSeatsFindResultDataKeywords = {
-  [ActiveSeatsFindResultFields.COLUMN_LAST_UPDATE]?: DataKeywords;
-  [ActiveSeatsFindResultFields.COLUMN_NUMBER]?: DataKeywords;
+  [ActiveSeatsFields.COLUMN_LAST_UPDATE]?: DataKeywords;
+  [ActiveSeatsFields.COLUMN_NUMBER]?: DataKeywords;
 };
 
 export type ActiveSeatsFindResultDataSortParameters = {
-  [ActiveSeatsFindResultFields.COLUMN_LAST_UPDATE]?: SortParameters;
-  [ActiveSeatsFindResultFields.COLUMN_NUMBER]?: SortParameters;
+  [ActiveSeatsFields.COLUMN_LAST_UPDATE]?: SortParameters;
+  [ActiveSeatsFields.COLUMN_NUMBER]?: SortParameters;
 };
 
 export type ActiveSeatsFindResultDataFiltersParameters = {
-  [ActiveSeatsFindResultFields.COLUMN_LAST_UPDATE]?: FiltersParameters;
-  [ActiveSeatsFindResultFields.COLUMN_NUMBER]?: FiltersParameters;
+  [ActiveSeatsFields.COLUMN_LAST_UPDATE]?: FiltersParameters;
+  [ActiveSeatsFields.COLUMN_NUMBER]?: FiltersParameters;
 };
 
-export class ActiveSeatsFindResult extends AbstractEntity<ActiveSeatsFindresultData> {
-  readonly #lastUpdate: string | null | undefined;
-  readonly #number: number | null | undefined;
-
-  public constructor(data: ActiveSeatsFindresultData) {
+export class ActiveSeatsFindResult extends AbstractActiveSeats {
```

Comment:
This class adds nothing to `AbstractActiveSeats` so maybe the abstract does not need to exist?
it's the exact same code than class `ActiveSeatsGetResult`
There should be only 1 version of the same code, please make only 1 class and use it from both places you need it.

Reply:
Done